.gitignore

```diff
@@ -1,3 +1,6 @@
+# SQLite database
+database.db
```

Comment:
Huh? I never had issues with mounting files in docker, even just a simple `/etc/localtime:/etc/localtime:ro` is a really common one, which is a mounting symlink file. I don't see why this would cause issues and I think it's kinda weird to put the database in an empty directory just for it, it's not like we'll ever be working with multiple database files.

Reply:
Probably windows

Reply:
Well, if you need something specific there, I did add `SQLITE_DATABASE_FILE` which you can override to change the path to the db file. Still quite odd that it wouldn't work though, maybe try it out and see if it's really an issue, if so, I can make it use a directory